### DISCUSSION TITLE
AstraDB/cassandra

### DISCUSSION THREAD

**Discussion by pedrocassalpacheco** [sqlalchemy/sqlalchemy#11137] [Ideas]
Greetings - I'd like to add support for AstraDB data API and CQL to sqlachemy. Can someone please point me in the right direction?

**Comment by CaselIT** [comment]
Hi,

ReadMe for dialect developers is at https://github.com/sqlalchemy/sqlalchemy/blob/main/README.dialects.rst
My suggestion would to look at it, a bit the internal dialect and the external ones to familiarize yourself with how that works. You can find a list of external dialects at https://docs.sqlalchemy.org/en/20/dialects/index.html

Feel free to send a PR to add that dialect to the list once you have something working

**Reply by pedrocassalpacheco** [reply]
Thanks. Will do.

**Reply by amotl** [reply]
Hi there. We are also looking for SQLAlchemy support for Apache Cassandra and its derivatives. 🙏

- https://github.com/bruin-data/ingestr/issues/14
